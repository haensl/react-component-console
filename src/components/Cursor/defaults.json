{
  "classes": {
    "blink": "Console-Cursor--blink",
    "element": "Console-Cursor",
    "write": "Console-Cursor--write"
  },
  "intervalMs": 400
}
