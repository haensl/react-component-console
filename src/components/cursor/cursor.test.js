import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';
import Cursor from './';

jest.useFakeTimers();

describe('Cursor', () => {
  describe('empty', () => {
    let tree;

    beforeEach(() => {
      tree = renderer.create(<Cursor />);
    });

    it('renders as expeted', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
