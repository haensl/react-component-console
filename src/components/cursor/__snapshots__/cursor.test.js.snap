// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cursor empty renders as expeted 1`] = `
<span
  class="console-cursor"
/>
`;
